### ISSUE TITLE
Traing KITTI result

### ISSUE THREAD

**Issue by CuongNguyen218** [jwchoi384/Gaussian_YOLOv3#45]
Hi @jwchoi384 ,
After training process with 60k iteration, I got this result. I see a big gap between result from your paper and my result in Car Moderate mAP.And I want to hear your idea about the gap? Is it normal ?
![image](https://user-images.githubusercontent.com/43350150/70901532-909c4900-202d-11ea-818a-da7aeb579a07.png)

**Comment by CuongNguyen218** [comment]
Can you provide train_kitti_list and test_kitti_list? i realize that my test_list have 21k box and yours have 17k box.

**Comment by jwchoi384** [comment]
@CuongNguyen218 
Hi, I uploaded KITTI train/validation list in my repo.
`train_kitti_3cls_list`
`valid_kitti_3cls_list`
As I mentioned in our paper, the KITTI training and validation sets are made by randomly splitting the training set in half.

**Comment by CuongNguyen218** [comment]
I want to hear your advice about my result @jwchoi384

**Comment by jwchoi384** [comment]
@CuongNguyen218 
I do not know the exact reason. It seems to be due to random training (ex. augmentation, resize, initialization etc.) or difference of training/validation list maybe.
Anyway, your results seems good.

**Comment by CuongNguyen218** [comment]
@jwchoi384 ,
![image](https://user-images.githubusercontent.com/43350150/71346084-a06ddd00-2599-11ea-98be-f7cd5ce62c7c.png)
I used your train-val spit and get this result on validation set at resolution 512 x 512 for car class. When i was training, I remove labels like tram, van, truck out of my dataset. How do you think about my result ? I realize that my result for car class is higher than what you get in your paper.

**Comment by jwchoi384** [comment]
@CuongNguyen218 
I also removed labels like tram, van, truck when training and validation.
In my opinion, performance may vary depending on random initialization, data augmentation, and system environment.
